examples.md

```diff
@@ -39,7 +53,7 @@ interfaces:
 The example includes 3 ethernet interfaces, two with static IPv4 and the 3rd
 with no IP and with the state set to down.
 
-```
+```yaml
```

Comment:
All examples are yaml, we do not need to mention it.

Reply:
It's for syntax highlight. I will remove it.

Reply:
Good catch @cathay4t, IMHO it would be better to actually add it everywhere If it improves the syntax hightlighting.

Reply:
Nice, I did not know it is for highlight.
It will be nice to see it everywhere in a new PR.